packages/core/docs/changelog/5684.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  description: 'remove filters sidebar to separate component and add ignore to nuxt config',
```

Comment:
```suggestion
  description: 'move filters sidebar to separate component ,
```
should part about removal be in CT changelog?

Reply:
Ok it should be.